### ISSUE TITLE
Prefer singular method names even for multi-valued fields

### ISSUE THREAD

**Issue by awead** [psu-stewardship/scholarsphere-4#192]
While fields such as descriptions, subtitles, and subjects are multivalued, their method names and json keys are not, ex:

`WorkVersion.new(subject: ["one", "two"])` stores json as `{"subject" : ["one", "two"] }`

This is consistent with the existing Scholarpshere implementation, with one exception: keywords. It is plural in Scholarsphere 4 but singular in Scholarsphere 3. Unless we feel strongly otherwise, we should make keywords a singular method, ex:

`WorkVersion.new(keyword: ["blah"])`

If we feel that a field's cardinality should be reflected in its method names and json representation, then we need to change all the other singular methods to plural ones. This latter solution will complicate migration since we'll need to translate key/value pairs.

### Option 1
* rename `keywords` method to `keyword`

### Option 2
* rename `description`, `subject`, `identifier`, etc. all to their plural equivalents

**Comment by srerickson** [comment]
This doesn't directly address the question, but why do we support multi-values for descriptions and subtitles?

**Comment by awead** [comment]
Because current Scholarsphere does. Also, from a linked data perspective, cardinality is always many. For example, you can enforce there to be a single triple for a given predicate. "Single valued" terms in ActiveFedora are actually arrays that we force to have only one element, but Fedora, for example, can't provide that kind of enforcement, so it's sort of a hack.

We could make multiple descriptions a single field of multiple paragraphs, assuming it's all plain text and no HTML or markdown. However, we did have use cases for multiple subtitles. They were, in theory, ordered, but it wasn't enforced properly.

**Comment by srerickson** [comment]
My preference would be to remove multi-values where possible for the sake of simplicity. The linked data perspective hasn't informed other design choices for SS4 so I don't think it should be a significant factor here. 

I'd like to learn more about the use cases for multiple subtitles.

**Comment by awead** [comment]
It looks like we've already made subtitles singular: https://github.com/psu-stewardship/scholarsphere/issues/948
I think we decided that if someone needed multiple subtitles they could use semicolons or some other kind of punctuation.

I would prefer option 1 because it's the least amount of change, i.e. changing 1 method name as opposed to 8, and it keeps in consistent with Scholarsphere 3. That way migration is much easier because the json key names stay the same. The display labels would be plural, so to the user there's no difference.
